ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -993,6 +1011,174 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P12S Pangenesis Collect',
+      type: 'GainsEffect',
+      netRegex: { effectId: pangenesisEffectIds },
+      condition: (data) => !data.pangenesisDebuffsCalled,
```

Comment:
```suggestion
      condition: (data) => !data.pangenesisDebuffsCalled && !data.isDoorBoss,
```

DF8/DF9 are used in p1 as well.